src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx

```diff
@@ -40,9 +73,11 @@ const backend = defineBackend({
 // extract L1 CfnUserPool resources
 const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
 // use CDK's `addPropertyOverride` to modify properties directly
-cfnUserPool.addPropertyOverride('Policies.PasswordPolicy.MinimumLength', 32);
+cfnUserPool.addPropertyOverride("Schema", [
```

Comment:
does this overwrite the existing schema (with `email`)?

Reply:
No, it just adds a custom attribute.